opennms-doc/guide-development/src/asciidoc/text/dev-guide.adoc

```diff
@@ -0,0 +1,101 @@
+
+This guide describes the requirements and the steps necessary in order to get started with the development of the OpenNMS project.
+
+=== Operating System / Environment
+
+To build/compile OpenNMS it is necessary to run a *nix system.
+You do not need to run it physically, a virtual machine is sufficient, but the choice is yours.
+We recommend one of the following:
+
+ * link:http://www.linuxmint.com/[Linux Minut] with Cinnamon Desktop environment
+ * link:http://ubuntu.com[Ubuntu Desktop]
+ * Mac OS X
+
+[NOTE]
+This documentation assumes that you chose a debian based desktop environment.
+
+=== Installation
+
+The next chapter describes the full setup of your environment in order to meet the pre-requirements.
+Simply follow these instructions, they may vary depending on your Operating System.
+
+[source, shell]
+----
+# add OpenNMS as repository to install icmp and such
+echo "deb http://debian.opennms.org stable main" > /etc/apt/sources.list.d/opennms.list
+echo "deb-src http://debian.opennms.org stable main" >> /etc/apt/sources.list.d/opennms.list
+# Add pgp key
+wget -O - http://debian.opennms.org/OPENNMS-GPG-KEY | apt-key add -
+
+# overall update
+apt-get update
+
+# install stuff
+apt-get install -y software-properties-common
+apt-get install -y git-core
+apt-get install -y nsis
+
+# install Oracle Java 8 JDK
+# this setup is based on: http://www.webupd8.org/2014/03/how-to-install-oracle-java-8-in-debian.html
+add-apt-repository -y ppa:webupd8team/java
+apt-get update
+apt-get install -y oracle-java8-installer
+apt-get install -y oracle-java8-set-default
+
```

Comment:
The Oracle Java 8 JDK seems to me redundant to the install guide. We should maintain just one.